### PR TITLE
chore(config/prow/arc): increment falco arm64 runners replicas.

### PR DESCRIPTION
**Pull request by FedeDP**
This way, we can support: PR CI on arm64 and master/release CI static arm64 binary.

### CODE REVIEW

**Comment by FedeDP** [comment]
/cc @maxgio92 

This leaves us with 15 nodes for arm64 drivers building pipeline (that means up to 30 pods). I think it is ok; we could also bump max autoscaling to 25, but i don't think it is worth it now.

**Comment by FedeDP** [comment]
The new implementation uses autoscaling based on the job queue for Falco repository.
/cc @maxgio92

File: config/prow/arc/arm-runner.yaml [hunk 1]
```diff
@@ -21,3 +20,18 @@ spec:
         operator: "Equal"
         value: "SingleAZ"
         effect: "NoSchedule"
+---
```
Comment (maxgio92) [inline config/prow/arc/arm-runner.yaml:23]: Interesting @FedeDP!

**Comment by maxgio92** [comment]
> /cc @maxgio92
> 
> This leaves us with 15 nodes for arm64 drivers building pipeline (that means up to 30 pods). I think it is ok; we could also bump max autoscaling to 25, but i don't think it is worth it now.

I think it's ok @FedeDP.
/lgtm

**Review by maxgio92 — APPROVED** [review]
/approve

**Comment by FedeDP** [comment]
Keep this on hold for now to avoid any disruptive change before the Falco release (even if we got plenty of time to revert or fix this :D )

**Comment by FedeDP** [comment]
No more needed since we are now using `actuated.dev` (https://actuated.dev/) provided runners.
/close

**Comment by FedeDP** [comment]
Indeed, once Falco CI is working with new runnes, we can remove the `arc` completely from test-infra.

**Pull request closed without merging**